src/data/scripts/pile/convert_the_pile.py

```diff
@@ -0,0 +1,59 @@
+from datasets import Dataset
+from argparse import ArgumentParser
+
+import zstandard as zstd
```

Comment:
Should we add `zstandard` module to `pyproject.toml`?

Reply:
zstandard is already in ```src/data/pyproject.toml```